portfolio/src/main/webapp/animation.js

```diff
@@ -0,0 +1,44 @@
+particles = [];
+
+function setup() {
+    createCanvas(screen.width, screen.height);
+}
+
+function draw() {
+    createParticles();
+    moveParticles();
+}
+
+function createParticles() {
+    for (let i = 0; i < 5; i++) {
+        let particle = new Particle();
+        particles.push(particle);
+    }
+}
+
+function moveParticles() {
+    for (let i = particles.length - 1; i >= 0; i--) {
+        particles[i].update();
+        particles[i].show();
+    }
+}
+
+class Particle {
+    constructor() {
+        this.x = screen.width / 2;
+        this.y = screen.height / 2;
+        this.vx = random(-2*Math.PI, 2*Math.PI);
```

Comment:
binary operators ask for whitespace as I understand it

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace